### DISCUSSION TITLE
Love it!

### DISCUSSION THREAD

**Discussion by larsabrasha** [tailwindlabs/tailwindcss#3207] [Kind words]
Using Tailwind saves so much time and brain power for me. Before I used Tailwind, I put lot of effort into making html and css the way you were supposed to do... But by using utility classes I can focus on the real problem I’m working on.

I have many good things to say about Tailwind. The best best thing is I don’t have to constantly come up with new class names. What a relief!

**Comment by simonswiss** [comment]
Thanks for sharing!

Yep - not having to name things all the time, by itself, makes Tailwind amazing 🎉
